app/components/plans/treatment.js

```diff
@@ -2,13 +2,19 @@
 import React from 'react';
 import {connect} from 'react-redux';
 //material-ui
-import {TableHeader,TableRowColumn,TableRow,Table, TableBody, FloatingActionButton} from 'material-ui';
+import {TableHeader,TableRowColumn,TableRow,Table, TableBody, FloatingActionButton, SelectField, MenuItem} from 'material-ui';
 import ContentAdd from 'material-ui/svg-icons/content/add';
 import {StepsTextField} from '../material-style';
 
 export default (props) => {
 
   const { exercise, treatment, addTreatment, treatmentHandler } = props;
+  const items = [
+  <MenuItem key={1} value={1} primaryText="None" />,
```

Comment:
Why don't we just make the values the actual enums? They are guaranteed to be unique and this way you don't need that extra `resistanceEnum` object